draft-ietf-dnsop-svcb-https.md

```diff
@@ -1521,7 +1526,7 @@ with SVCB.
 
 # Security Considerations
 
-SVCB/HTTPS RRs are intended for distribution over untrusted
+SVCB/HTTPS RRs permit distribution over untrusted
```

Comment:
just a couple paragraphs prior we said "intended for use over a privacy-preserving DNS transport", which is not exactly an untrusted channel (though it's also not a fully trusted channel).

Reply:
makes sense